### PR TITLE
namespace, class, and method cleanup for generated docs

### PR DESCRIPTION
**Pull request by marek**
- renamed menu/menuitem files to fall under blackberry_ui_
- fixed overload for blackberry.push.Data.prototype.getHeaderField() so it does not get clobbered as a duplicate
- removed duplicate blackberry.invoke.BrowserArguments

### CODE REVIEW

File: api/blackberry_push_Data.js [hunk 1]
```diff
@@ -120,16 +113,16 @@ blackberry.push.Data.DECLINE_USERRFS = 5;
 * @returns {String}
 * @BB50+
 */
-blackberry.push.Data.prototype.getHeaderField = function(field) { };
-
 /**
 * Returns the value of a header field. 
+* @name blackberry.push.Data.prototype.getHeaderField^2
```
Comment (jeffheifetz) [inline api/blackberry_push_Data.js:121]: This should not be necessary. Removing the duplicated set of comments and adding an @function tag should fix the issue.

File: api/blackberry_push_Data.js [hunk 2]
```diff
@@ -120,16 +113,16 @@ blackberry.push.Data.DECLINE_USERRFS = 5;
...
-blackberry.push.Data.prototype.getHeaderField = function(field) { };
-
 /**
 * Returns the value of a header field. 
+* @name blackberry.push.Data.prototype.getHeaderField^2
 * @param {Number} field This parameter is the integer index of the header to retrieve.
 * @returns {String}
 * @BB50+
 */
 blackberry.push.Data.prototype.getHeaderField = function(field) { };
```
Comment (jeffheifetz) [inline api/blackberry_push_Data.js:126]: While you're fixing the file, if you remove the prototype from the function names, they render properly with # characters

File: api/blackberry_push_Data.js (hunk 1 above)
Comment (marek) [inline api/blackberry_push_Data.js:121]: They are not duplicate, but overloaded.  One takes a string, the other takes a number.

Comment (jeffheifetz) [inline api/blackberry_push_Data.js:121]: Nice catch, we'll have to find a uniform way to document this

File: api/blackberry_push_Data.js (hunk 2 above)
Comment (jeffheifetz) [inline api/blackberry_push_Data.js:126]: I never realized this was to differentiate instance and static children. I guess we should just leave it

**Comment by jeffheifetz** [comment]
Still have the empty namespace for blackberry.ui

**Comment by jeffheifetz** [comment]
I think its great to go

**Pull request merged**